src/Button/Button.jsx

```diff
@@ -30,57 +47,36 @@ const Button = ({
     {...rest}>
     {loading ? (
       <ActivityIndicator
-        colors={spinnerVariants[variant]}
+        colors={spinnerColor}
```

Comment:
What if the button has the same background as the spinerColor? Shouldn't we use spinner variants here?